boa/verifiers.py

```diff
@@ -166,6 +185,24 @@ def verify(
         address=contract.address,
         solc_json=bundle,
         contract_name=contract.contract_name,
-        license_type=license_type,
+        constructor_calldata=contract.ctor_calldata,
         wait=wait,
+        **kwargs,
     )
+
+
+def _wait_until(
```

Comment:
the indirection here is extremely confusing.

Reply:
why not just make it part of the base contract verifier?

Reply:
While it is in the verifiers file, it is definitely not specific for contract verification.
Since it doesn't use anything from `self` I didn't see the point of keeping it in a class